docs/source/nodeoperations/restore.md

```diff
@@ -2,88 +2,223 @@
 
 This procedure will describe how to restore from backup taken using [Scylla Manager](../manager.md) to a fresh **empty** cluster of any size.
 
-First identify to which snapshot you want to restore. To get list of available snapshot execute following command on Scylla Manager Pod.
-```bash
+:::{caution}
+Due to a [bug](https://github.com/scylladb/scylla-manager/issues/3679) in Scylla Manager not supporting ScyllaClusters having both non-TLS and TLS CQL ports open, we have to disable TLS certificate management
+in Scylla Operator before you restore your cluster. Once bug is fixed, this is no longer required.
+
+To disable TLS certificate management in Scylla Operator add `--feature-gates="AutomaticTLSCertificates=false"` to Scylla Operator deployment.
+
+```console
+kubectl -n scylla-operator patch deployment/scylla-operator --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--feature-gates="AutomaticTLSCertificates=false"}]'
+```
+
+:::
+
+In this example our ScyllaCluster from which we took backup is called `source`. Backup will be restored into the ScyllaCluster called `target`.
+
+::::{tab-set}
+:::{tab-item} Source ScyllaCluster
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: source
+  namespace: scylla
+spec:
+  agentVersion: 3.2.5
+  version: 5.4.1
+  developerMode: true
+  backups:
+  - name: foo
+    location:
+    - s3:source-backup
+    keyspace:
+    - '*'
+  datacenter:
+    name: us-east-1
+    racks:
+    - name: us-east-1a
+      members: 1
+      storage:
+        capacity: 1Gi
+      resources:
+        limits:
+          cpu: 1
+          memory: 1Gi
+```
+:::
+:::{tab-item} Target ScyllaCluster
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: target
+  namespace: scylla
+spec:
+  agentVersion: 3.2.5
+  version: 5.4.1
+  developerMode: true
+  datacenter:
+    name: us-east-1
+    racks:
+    - name: us-east-1a
+      members: 1
+      storage:
+        capacity: 1Gi
+      resources:
+        limits:
+          cpu: 1
+          memory: 1Gi
+```
+:::
+::::
+
+Make sure our target cluster is already registered in Scylla Manager, to get list of all registered clusters execute following command in Scylla Manager Pod:
```

Comment:
```suggestion
First, make sure your target cluster is already registered in Scylla Manager. To get a list of all registered clusters, execute the following command in the Scylla Manager Pod:
```

Reply:
On a second thought though, I wouldn't say `run the following command in X`, but post a `kubectl exec` command against the manager deployment in each of those steps, .i.e. `kubectl exec -n scylla-manager -it deployment.apps/scylla-manager -- sctool cluster list`. I think it's more bulletproof.

Reply:
done

Reply:
Some weird hybrid came out of that now: `Make `target` cluster is already registered in Scylla Manager`